js/about/certerror.js

```diff
@@ -49,12 +63,30 @@ class CertErrorPage extends React.Component {
         <span data-l10n-id='certErrorText'></span>&nbsp;
         <span className='errorUrl'>{this.state.url || ''}</span>
         <span className='errorText'>{this.state.error || ''}</span>
+        {this.state.cert
+          ? (<div>
+            <span className='certErrorText'>{'Issued To'}</span>
+            <span className='errorText'>{'Common Name (CN): '}</span>
+            <span className='errorText'>{'Organization (O): '}</span>
+            <span className='errorText'>{'Organization Unit(OU): '}</span>
+            <span className='errorText'>{'Serial Number: '}</span>
+            <span className='certErrorText'>{'Issued By'}</span>
+            <span className='errorText'>{'Common Name (CN): ' + this.state.cert.issuerName}</span>
```

Comment:
@diracdeltas do we have to submit a issue which is sending the entire cert object via IPC causes a renderer crash?
And do we still need a link for user to download raw certificate after showing more [detail](https://github.com/brave/browser-laptop/pull/2428#issuecomment-232114982)